UnitTests/Schemas/IFC4X1/Comments/Data/csg-primitive-commented.ifc

```diff
@@ -0,0 +1,54 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION((''),'2;1');
+FILE_NAME('','2019-03-20T15:56:21',(''),(''),'BuildingSmart IfcKit by Constructivity','IfcDoc 12.0.0.0','');
+FILE_SCHEMA(('IFC4x1'));
+ENDSEC;
+
+DATA;
+
+#1= IFCPROJECT('0xScRe4drECQ4DMSqUjd6d',#2,'proxy with CSG',$,$,$,$,(#3),#4);
+#2= IFCOWNERHISTORY(#6,#7,$,.ADDED.,1320688800,$,$,1320688800);
+#3= IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,1.0E-05,#8,$);
+#4= IFCUNITASSIGNMENT((#10,#11));
+#6= IFCPERSONANDORGANIZATION(#12,#13,$);
+#7= IFCAPPLICATION(#13,'1.0','IFC text editor','ifcTE');
+#8= IFCAXIS2PLACEMENT3D(#14,$,$);
+#9= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body','Model',0,$,$,$,#3,$,.MODEL_VIEW.,$);
+#10= IFCSIUNIT($,.LENGTHUNIT.,.MILLI.,.METRE.);
+#11= IFCCONVERSIONBASEDUNIT(#16,.PLANEANGLEUNIT.,'degree',#17);
+#12= IFCPERSON($,'Liebich','Thomas',$,$,$,$,$);
+#13= IFCORGANIZATION($,'buildingSMART International',$,$,$);
+#14= IFCCARTESIANPOINT((0.,0.,0.));
+#15= IFCSHAPEREPRESENTATION(#9,'Body','CSG',(#18));
+#16= IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#17= IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453293),#20);
+#18= IFCCSGSOLID(#21);
+#19= IFCPRODUCTDEFINITIONSHAPE($,$,(#15));
+
+// This time this is correct!

```

Comment:
Add a comment at the end of some line with content as well.
Sth like #77 = ENTITY( content ); // This is a comment